cve_bin_tool/cli.py

```diff
@@ -290,8 +289,11 @@ def output_cves(outfile, modules, include_details=False):
                 writer.writerow(row)
 
 
-def main(argv=sys.argv, outfile=sys.stdout):
+def main(argv=None, outfile=sys.stdout):
     """ Scan a binary file for certain open source libraries that may have CVEs """
+    if argv is None:
+        argv = sys.argv
```

Comment:
What's the purpose of this change?

Reply:
I see your comment says this makes it not mutable, what do you mean by this? My understanding is that this changes preserves mutability. If we wished to make changes to `argv` not change `sys.argv`, we'd have to use `copy.deepcopy`

Reply:
This is related to `sys.argv` being passed as a default argument in the function definition (on line 292). 

```python
def main(argv=sys.argv, outfile=sys.stdout):
    ...
```

It's not recommended to pass mutable objects as default values since the latest passed value is preserved on subsequent function calls, and can cause unintended effects. [(ref)](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)

Reply:
Ah okay I see what you mean, cool!